packages/client/src/__tests__/integration/happy/full-text-search-mysql/test.ts

```diff
@@ -0,0 +1,291 @@
+import path from 'path'
+import { generateTestClient } from '../../../../utils/getTestClient'
+import { tearDownMysql } from '../../../../utils/setupMysql'
+import { migrateDb } from '../../__helpers__/migrateDb'
+
+const testIf = (condition: boolean) => (condition ? test : test.skip)
+
+// @ts-ignore trick to get typings at dev time
+import type { PrismaClient } from './node_modules/@prisma/client'
+
+let prisma: PrismaClient
+const baseUri = process.env.TEST_MYSQL_URI
+describe('full-text-search (mysql)', () => {
+  beforeAll(async () => {
+    process.env.TEST_MYSQL_URI += '-full-text-search'
+    await tearDownMysql(process.env.TEST_MYSQL_URI!)
+    await migrateDb({
+      connectionString: process.env.TEST_MYSQL_URI!,
+      schemaPath: path.join(__dirname, 'schema.prisma'),
+    })
+    await generateTestClient(__dirname)
+    const { PrismaClient } = await require('./node_modules/@prisma/client')
+
+    prisma = new PrismaClient()
+
+    await prisma.user.create({
+      data: {
+        email: 'email1@email.io',
+        name: 'John Smith',
+      },
+    })
+
+    await prisma.user.create({
+      data: {
+        email: 'email2@email.io',
+        name: 'April ONeal',
+      },
+    })
+
+    await prisma.user.create({
+      data: {
+        email: 'email3@email.io',
+        name: 'John Pearl',
+      },
+    })
+  })
+
+  afterAll(async () => {
+    await prisma.user.deleteMany()
+    await prisma.$disconnect()
+    process.env.TEST_MYSQL_URI = baseUri
+  })
+
+  /**
+   * Test search with the `+` operator
+   */
+  test('findMany with +', async () => {
+    const result = await prisma.user.findMany({
+      where: {
+        name: {
+          search: '+John +Smith',
+        },
+      },
+    })
+
+    expect(result).toMatchInlineSnapshot(`
+      Array [
+        Object {
+          email: email1@email.io,
+          id: 1,
+          name: John Smith,
+        },
+      ]
+    `)
+  })
+
+  /**
+   * Test search with OR
+   */
+  test('findMany with OR', async () => {
+    const result = await prisma.user.findMany({
+      where: {
+        name: {
+          search: 'John April',
```

Comment:
Oh so `OR` is a blank space?

Reply:
So https://dev.mysql.com/doc/refman/8.0/en/fulltext-boolean.html

Reply:
Yes. Its a lot more basic thank what Postgres offers